src/views/menus/AccountMenu.tsx

```diff
@@ -386,6 +374,46 @@ export const AccountMenu = () => {
   );
 };
 
+const DydxDerivedAddress = ({
+  evmAddress,
+  solAddress,
+  dydxAddress,
+}: {
+  evmAddress?: string;
+  solAddress?: string;
+  dydxAddress?: string;
```

Comment:
nit: would a better typing be:

```
type DerivedSolAddress = {
  dydxAddress: string;
  solAddress: string;
}

type DerivedEvmAddress = {
  dydxAddress: string;
  evmAddress: string;
}

const DydxDerivedAddress = (props: DerivedSolAddress | DerivedEvmAddress) => {...}
```

Reply:
re: the translation one, i think if we do that, then the blockchain name wouldn't be included in the translation, right?